Source/WebCore/platform/graphics/displaylists/DisplayList.h

```diff
@@ -65,9 +69,12 @@ class DisplayList {
     WEBCORE_EXPORT String asText(OptionSet<AsTextFlag>) const;
     void dump(WTF::TextStream&) const;
 
+    const OptionSet<RecordReplayOption>& applyItemOptions() const { return m_options; }
```

Comment:
`applyItemOptions()` -> `replayOptions()`?

Reply:
Indeed, applyItemOptions() is a left-over... However since `recordAndReplayOptions()` sounds awkward, I'll go with your `ReplayOption` + `replayOptions()` suggestion.